### ISSUE TITLE
[Question] vscode-extension-telemetry npm module

### ISSUE THREAD

**Issue by solsu01** [eclipse-theia/theia#6330]
### Description
Am I able to use vscode-extension-telemetry npm module to perform telemetry from Theia? 
const TelemetryReporter = require('vscode-extension-telemetry'); will that work?

For reference: https://code.visualstudio.com/docs/getstarted/telemetry

**Comment by akosyakov** [comment]
I don't think so, i'm not sure that we want to support telemetry out of the box.

**Comment by joazvsoares** [comment]
Hey, I'm interested in telemetry support too.

@solsu01 have you made any progress about it?

**Comment by solsu01** [comment]
@joazvsoares nope

**Comment by hannescmarits** [comment]
YES, i think Theia should provide the possibility to support telemetry. 
Telemetry is a feature which helps to prioritize. 
Helps to see what is used, and how the software is used. 
Yes, the topic is viewed critically, but data about user behavior is just very helpful.
